sc2/unit.py

```diff
@@ -357,6 +357,14 @@ def calculate_speed(self, upgrades: Set[UpgradeId] = None) -> float:
             speed *= SPEED_ALTERING_BUFFS.get(buff, 1)
         return speed
 
+    @property
+    def distance_per_step(self) -> float:
+        return (self.real_speed/22.4) * self._bot_object.client.game_step
```

Comment:
Some information in a docstring is desired - when should one use this property?

Reply:
Added